test/database/cipher_test.go

```diff
@@ -0,0 +1,20 @@
+package database
```

Comment:
either pull out fakeCipher into cipher.go or rename this file inmemory_test.go

Reply:
done

Reply:
done